ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install.cmake.in

```diff
@@ -30,7 +30,7 @@ function(ament_cmake_symlink_install_directory)
   endif()
 
   # default pattern to include
-  if("${ARG_PATTERN} " STREQUAL " ")
+  if(NOT ARG_PATTERN)
```

Comment:
In this particular case, that's probably fine, but I didn't consider closely whether or not this less strict comparison will be a problem in each case where you moved away from `STREQUAL`. I suppose we can just merge these and deal with issues if they arise. I was just curious if you considered this while changing things throughout the pr's.